sale_product_set_variants/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+Sale Product Set Variants
+================
+
+Extension of the sale product set module to make it work with variants
+
+Usage
+=====
+
+* Activate the product.variants in sale -> settings
+TODO: complete usgae
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/sale-workflow/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/sale-workflow/issues/new?body=module:%20sale_product_set%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

Comment:
simplify this link, see https://github.com/OCA/maintainer-tools/tree/master/template/module